src/renderer/pages/Staking/Operations/components/Validators/Validators.test.tsx

```diff
@@ -14,20 +23,14 @@ jest.mock('@renderer/app/providers', () => ({
 }));
 
 jest.mock('@renderer/entities/staking', () => ({
+  ...jest.requireActual('@renderer/entities/staking'),
   useValidators: jest.fn().mockReturnValue({
     getMaxValidators: jest.fn().mockReturnValue(6),
-    getValidatorsWithInfo: jest.fn().mockResolvedValue({
-      '5C556QTtg1bJ43GDSgeowa3Ark6aeSHGTac1b2rKSXtgmSmW': {
-        address: '5C556QTtg1bJ43GDSgeowa3Ark6aeSHGTac1b2rKSXtgmSmW',
-        totalStake: '84293898648764293',
-        ownStake: '27425315022592146',
-        blocked: false,
-      },
-    }),
   }),
   useEra: jest.fn().mockReturnValue({
     subscribeActiveEra: jest.fn().mockImplementation((api: any, eraCallback: any) => eraCallback(1)),
   }),
+  useValidatorsMap: jest.fn().mockImplementation(() => validatorsMap),
```

Comment:
I guess this should work as well
`useValidatorsMap: jest.fn().mockResolvedValue(validatorsMap)`

Reply:
No, it doesn't. It throws error abount using validatorsMap const before initialisation